src/components/style/form.css

```diff
@@ -0,0 +1,44 @@
+.form-section {
```

Comment:
- [ ] OPTIONAL
You can improve your design further by applying the fonts used in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)

**Original design:**
![image](https://user-images.githubusercontent.com/12745474/185769922-d8cda7d5-8f2f-420b-ac2c-26431dc3b23c.png)